dusty/compiler/nginx/__init__.py

```diff
@@ -1,7 +1,7 @@
-from ...constants import NGINX_MAX_FILE_SIZE
+from ...constants import NGINX_MAX_FILE_SIZE, VM_IP_FROM_DOCKER
 
 def _nginx_proxy_string(port_spec):
-    return "proxy_pass http://{}:{};".format(port_spec['boot2docker_ip'], port_spec['proxied_port'])
+    return "proxy_pass http://{}:{};".format(VM_IP_FROM_DOCKER, port_spec['proxied_port'])
```

Comment:
so nginx lives inside a container how is this getting out to the virtual machine?  Or is this, the ip of the outside world if I am in a container? if so that is is pretty nifty?